.github/workflows/ci-go.yml

```diff
@@ -71,3 +71,18 @@ jobs:
 
       - name: CLI integration tests
         run: make bin/ak test-cli
+
+  test-runs:
+    name: Run tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+
+      - name: CLI integration tests
+        run: make bin/ak test-runs
```

Comment:
I prefer to have a single `ci` Makefile rule and have the logic of what to run in CI there instead of this YAML file.

Reply:
looking into it - might change in a future pr.